docs/operations/scheduler.md

```diff
@@ -0,0 +1,128 @@
+Server Version: version.Info{Major:"1", Minor:"22", GitVersion:"v1.22.13", GitCommit:"a43c0904d0de10f92aa3956c74489c45e6453d6e", GitTreeState:"clean", BuildDate:"2022-08-17T18:23:45Z", GoVersion:"go1.16.15", Compiler:"gc", Platform:"linux/amd64"}
+```
+
+Pay attention to the `Server` line. 
+In this case, the Kubernetes version is `v1.22.13`.
+You have to choose the appropriated Kubernetes version and [build the Kubernetes project](https://kubernetes.io/docs/tasks/extend-kubernetes/configure-multiple-schedulers/#package-the-scheduler):
+
+```shell
+$ cd kubernetes
+$ git checkout v1.22.13
+$ make
+```
+
+Then, you can follow the configuration steps described [here](https://kubernetes.io/docs/tasks/extend-kubernetes/configure-multiple-schedulers/).
+Additionally, you must  allow `storageclasses` to use the verbs: `watch`, `list` and `get`, you can add the following at the end of the 
+`clusterrole system:kube-scheduler`:
+
+```yaml
+- apiGroups:                                                                                                   
+  - storage.k8s.io                                                                                             
+  resources:                                                                                                   
+  - storageclasses                                                                                             
+  verbs:                                                                                                       
+  - watch                                                                                                      
+  - list                                                                                                       
+  - get 
+```
+
+
+## Scheduling VMs with the Custom Scheduler
+
+The second scheduler should be up and running. You can check it with:
+
+```shell
+$ kubectl get all -n kube-system
+```
+
+A deployment called `my-scheduler` have to be up and running if everything is properly configured.
+
+In order to launch VM/VMIs using the `my-scheduler`, you need to add `spec.SchedulerName: my-scheduler` to the VM
+definition. Here is an VM definition example:
+
+```yaml
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  labels:
+    kubevirt.io/vm: vm-fedora-vm
+  name: vm-fedora
+spec:
+  running: true
+  template:
+    metadata:
+      labels:
+        kubevirt.io/domain: vm-fedora
+        kubevirt.io/vm: vm-fedora
+    spec:
+      schedulerName: my-scheduler
+      domain:
+        cpu:
+          cores: 1
+          sockets: 1
+          threads: 1
+        devices:
+          disks:
+            - name: containerdisk
+              disk:
+                bus: virtio
+            - name: cloudinitdisk
+              disk:
+                bus: virtio
+          interfaces:
+            - name: default
+              masquerade: {}
+          rng: {}
+        resources:
+          requests:
+            memory: 1Gi
+      networks:
```

Comment:
Omit the `networks` configuration for the sake of simplicity? I don't think it adds any value to this example.